report_xls/__openerp__.py

```diff
@@ -26,8 +26,7 @@
     'author': "Noviat,Odoo Community Association (OCA)",
     'website': 'http://www.noviat.com',
     'category': 'Reporting',
-    'description': """
-Excel report engine
+    'description': """Excel report engine
```

Comment:
Simply remove the description key. This is not mandatory anymore with the README file.
(even more this becomes redundant and difficult to keep in sync)